core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -336,6 +279,10 @@ contract Liquidatable is PricelessPositionManager {
                 (msg.sender == liquidation.sponsor)
         );
 
+        // Settles the liquidation if necessary.
+        // Note: this will fail if the price has not resolved yet.
```

Comment:
To be clearer, just add a comment that _settle is only done when `State == PendingDispute` and is only done once

Reply:
Will do.

Reply:
Done.